echemdb/data/cv/entry.py

```diff
@@ -100,6 +100,38 @@ def __getitem__(self, name):
         """
         return Descriptor(self.package.descriptor)[name]
 
+    def citation(self, render='text'):
+        r"""
+        Returns a formatted reference for the entry's bibliography either as:
+    
+        J. Doe, et al., Journal Name, volume (YEAR) page, "Title"
+
+        Rendering default is plain text 'text', but can be changed to any format 
+        supported by pybtex, such as markdown 'md', 'latex' or 'html'. 
+
+        EXAMPLES::
+
+            >>> entry = Entry.create_examples()[0]
+            >>> entry.citation(render='text')
+            'O.Alves et al., Physical Chemistry Chemical Physics, 13 (2011) 6010–6021, "Electrochemistry at Ru(0001) in a flowing CO-saturated electrolyte—reactive and inert adlayer phases"'
+
+        """
+        from pybtex.textutils import abbreviate
+        from pybtex.style.template import field, optional, tag, words, join
+        from pybtex.style.formatting.unsrt import Style
+
+        firstname_inititial = abbreviate(self.bibliography.persons['author'][0].first_names[0])
```

Comment:
Apparently, this does not work properly when the author's name contains latex encoding. (see the briega_martos_2021 ... entries when they are rendered. 
This was fixed when I used the 'pylatexenc' module at this point, which renders this latex string to text. Unless someone has a better idea, I would change this back.

Reply:
The problem is that abbreviate turns the name into a string and then pybtex does not know anymore that this is LaTeX.